convert_util.c

```diff
@@ -108,7 +111,8 @@ convert_parse_tlvs(const uint8_t *buff, size_t buff_len,
 
 			opts->flags		|= CONVERT_F_CONNECT;
 			opts->remote_addr	= conv_connect->remote_addr;
```

Comment:
no need to do anything for the addr and the flags?

Reply:
As discussed offline, I will change struct convert_opts to use a sockaddr_in6. Closing this.
Will squash the zero-buff check with unit test uncovered this.